modules/ROOT/pages/fd-connection-view-audit-logs.adoc

```diff
@@ -0,0 +1,24 @@
+= View Audit Logs for Connections That are Shared
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords:
+
+:imagesdir: ../assets/images
+
+Although only the owner of a shared connection can edit or delete it, someone might illicitly acquire the credentials and edit or delete one or more shared connections. Someone also might maliciously use another user's credentials to create a shared connection and siphon data to an unapproved system.
+
+To help you identify such scenarios, Flow Designer creates an audit-log message for every creation, change to, and deletion of a shared connection within your business group. Each message includes the following information:
+
+* The time and date of the action
+* The product in which the action was taken (i.e., Flow Designer)
+* The type of change, which is always "Connection"
```

Comment:
I'm not sure if the "type of change" is the best definition. I think it's more like "type of audit object", which in this case is a Connection. What do you think?

Reply:
I changed this to "type of object audited".